src/pages/en/guides/cms/wordpress.md

```diff
@@ -2,12 +2,222 @@
+
+A large quantity of data is available to you via this API, so you may wish to only fetch certain fields. You can restrict your response by adding the `_fields` paramater to the API URL, for example: `[YOUR-SITE]/wp/v2/posts?_fields=author,id,excerpt,title,link` 
+
+The API can also return content related to your post, such as a link to the parent post, or to comments on the post. You can add the `_embed` parameter to the API URL (e.g. `[YOUR-SITE]/wp/v2/posts?_embed`) to indicate to the server that the response should include these embedded resources.
+
+## Using WordPress Data in Astro
+
+This example fetches data from the public WordPress API of [https://norian.studio/dinosuars/](https://norian.studio/dinosaurs/). This WordPress site stores information about individual dinosaurs under the `dinos` route, just as a blog would store individual blog posts under the `posts` route.
+
+This example shows how to reproduce this site structure in Astro: an index page that lists dinosaurs with links to dynamically-generated individual dinosaur pages.
+
+:::note
+To use [Custom Post Types (CPT)](https://learn.wordpress.org/lesson-plan/custom-post-types/) in your WordPress API (not just `post` and `page`), you will have to [configure them in your WordPress dashboard](https://stackoverflow.com/questions/48536646/how-can-i-get-data-from-custom-post-type-using-wp-rest-api) or [add REST API Support For Custom Content Types](https://developer.wordpress.org/rest-api/extending-the-rest-api/adding-rest-api-support-for-custom-content-types/) in WordPress.
+
+This example fetches data from a WordPress site whose content types have already been configured and exposed to the REST API.
+:::
+
+### Writing WordPress API calls
+
+You may find it helpful to create a file `src/lib/api.js` to store the functions that will make the necessary API calls.
+
+```ini title="Project Structure" {2-3}
+├── src/
+├── lib/
+│    └── api.js
+├── pages/
+│    └── index.astro
+│    └── dinos/
+│         └── [slug].astro
+├── astro.config.mjs
+└── package.json
+```
+
+In `src/lib/api.js`, you can export the `fetch` functions to send requests to the WordPress backend. This file defines a `fetchAPI()` function, as well as functions that use it to fetch all posts, and just a single post.
+
+```js title="lib/api.js"
+const API_URL = "https://norian.studio/wp-json/wp/v2/";
+
+// Get post data by API URL and given path
+export async function fetchAPI(path) {
+    const res = await fetch(`${API_URL}${path}`);
+    const json = await res.json();
+    return json;
+}
+
+export async function getPosts() {
+    let posts = await fetchAPI('dinos?per_page=50&_embed');
+    return posts;
+}
+
+// Filters the posts by slug, returns the matching post
+export async function getPost(slug) {
+    let posts = await fetchAPI(`dinos?slug=${slug}&_embed`);
+    return posts[0];
+}
+```
+
+### Index page
```

Comment:
We can use a more descriptive title here. Again readers will usually look up the TOC and just click on the section they are interested in.

```suggestion
### Displaying a list of dinosaurs
```

Reply:
Agreed! Let's do this!